lib/cli/commands/listpeers.ts

```diff
@@ -1,11 +1,74 @@
 import { callback, loadXudClient } from '../command';
 import { Arguments } from 'yargs';
-import { ListPeersRequest } from '../../proto/xudrpc_pb';
+import colors from 'colors/safe';
+import Table, { HorizontalTable } from 'cli-table3';
+import { generateHeaders } from '../utils';
+import { ListPeersRequest, ListPeersResponse, Peer } from '../../proto/xudrpc_pb';
+
+const HEADERS = [
+  'address',
+  'public key',
+  'pairs list',
```

Comment:
I don't think we actually need "list" here. Just "pairs" is more concise and still descriptive enough I think.